pom.xml

```diff
@@ -632,7 +632,7 @@
                 <artifactId>wildfly-appclient</artifactId>
                 <version>${ee.maven.version}</version>
             </dependency>
-            
+
```

Comment:
Hi @Ashpan 

Please adjust your editor config so it doesn't make this kind of whitespace change. Mass whitespace changes increase the chances of merge conflicts between PRs and tend to obscure the important parts of a change.